src/content/h1/map/readme.md

```diff
@@ -5,25 +5,36 @@ img: map-icon.jpg
 thanks:
   - to: Masterz1337
     for: Context on OpenSauce capabilities
+  - to: Jakey
+    for: Tag data limit info
 ---
 
 A **map**, also known as a **cache file**, is a bundle of compiled [tags][] which can be loaded and used by [Halo][h1]. With the exception of _resource maps_, each map represents a playable campaign, multiplayer level, or main menu.
 
-As the name _cache_ suggests, these files are intended to be loaded directly into Halo's memory at a fixed address with no further processing of data needed. The files contain pre-calculated pointers
+Halo loads these files directly into memory at a fixed address with no further processing of data needed, which makes map load times as short as possible. Tags within a map cache can contain pre-calculated pointers and matrices, and other fields which are needed by the game.
 
-Maps are found in Halo's `maps` directory. Maps in subdirectories are not loaded by the game. Mods like Chimera and HAC2 store downloaded maps in a separate location and force the game to load them regardless.
+Maps are found in Halo's `maps` directory. Maps in subdirectories are not loaded by the game. Mods like Chimera and HAC2 store [downloaded](#halonet) maps in a separate location and force the game to load them regardless.
+
+# Limits
+Maps have a limit of 23 MiB of tag data (raised to 31 MiB in MCC). This does not include the raw BSP data or bitmaps, just tag fields. [Tool][] will output the used tag space when compiling a map, for example:
```

Comment:
In Custom Edition BSP are loaded at the end of the tag memory. So, at any given time the tag space is shared with one bsp